doc/Administration-guide/index.md

```diff
@@ -5,4 +5,21 @@ permalink: /:path/
 sort_idx: 30
 ---
 
-Guide for the platform administrators.
\ No newline at end of file
+{% include variables.md %}
+
+The intention of this guide is to introduce Kaa platform administrative tasks for you.
+
+# Sections overview
+
+This guide provide the overview of [key system components]({{root_url}}Administration-guide/System-components-overview/) from the administration point of
+view. Helps with [planning your deployment]({{root_url}}Administration-guide/System-installation/Planning-your-deployment/), provides an overview of
+[Kaa platform installation]({{root_url}}Administration-guide/System-installation/) on a
+[single Linux node]({{root_url}}Administration-guide/System-installation/Single-node-installation/) and administration of
+[Kaa cluster on Linux nodes]({{root_url}}Administration-guide/System-installation/Cluster-setup/). After installation
```

Comment:
[cluster of Linux nodes]({{root_url}}Administration-guide/System-installation/Cluster-setup/).